scripts/provision.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
Move all content in `rhel-edge-84-virt-test.sh` into this file, and do not need `rhel-edge-84-virt-test.sh`.

Reply:
rhel-edge-84-virt-test.sh will checkout git branch
provision.sh is a file that required by the ostree-ng.sh in the git branch.

so, you mean we only do checkout in rhel-edge-84-virt-test.sh, and move all other setup script to provision.sh?